simulation/traffic_simulator/include/traffic_simulator/entity/entity_base.hpp

```diff
@@ -140,9 +140,15 @@ class EntityBase
   /*   */ void updateStandStillDuration(const double step_time);
   virtual const autoware_vehicle_msgs::msg::VehicleCommand getVehicleCommand();
 
+  const boost::optional<openscenario_msgs::msg::EntityStatus> getEntityStatusBeforeUpdate() const
+  {
+    return status_before_update_;
+  }
+
```

Comment:
This file is intentionally formatted so that the member function names are aligned.
Rewrite as follows according to the format of other member functions.

```
  virtual auto getVehicleCommand() -> const autoware_vehicle_msgs::msg::VehicleCommand;

  /*   */ auto getEntityStatusBeforeUpdate() const -> const boost::optional<openscenario_msgs::msg::EntityStatus>
  {
    return status_before_update_;
  }
```

Reply:
Also, the member functions of EntityBase are arranged in lexicographic order, so follow them.